TextToSpeech/impl/TTSSpeaker.cpp

```diff
@@ -652,8 +699,8 @@ void TTSSpeaker::speakText(TTSConfiguration config, SpeechData &data) {
         m_currentSpeech = &data;
 
         g_object_set(G_OBJECT(m_source), "location", constructURL(config, data).c_str(), NULL);
-        // PCM Sink seems to be accepting volume change before PLAYING state
         g_object_set(G_OBJECT(m_audioSink), "volume", (double) (data.client->configuration()->volume() / MAX_VOLUME), NULL);
```

Comment:
should this be m_audioVolume?

Reply:
Corrected the same and pushed the changes